proposed/phpdoc.md

```diff
@@ -0,0 +1,2214 @@
+PSR-5: PHPDoc
+=============
+
+## Table Of Contents
+
+- [1. Introduction](#1-introduction)
+- [2. Conventions Used In This Document](#2-conventions-used-in-this-document)
+- [3. Definitions](#3-definitions)
+- [4. Basic Principles](#4-basic-principles)
+- [5. The PHPDoc Format](#5-the-phpdoc-format)
+  - [5.1. Summary](#51-summary)
+  - [5.2. Description](#52-description)
+  - [5.3. Tags](#53-tags)
+    - [5.3.1. Tag Name](#531-tag-name)
+    - [5.3.2. Tag Specialization](#532-tag-specialization)
+    - [5.3.3. Tag Signature](#533-tag-signature)
+  - [5.4. Inline PHPDoc](#54-inline-phpdoc)
+  - [5.5. Examples](#55-examples)
+- [6. Inheritance](#6-inheritance)
+  - [6.1. Class Or Interface](#61-class-or-interface)
+  - [6.2. Function Or Method](#62-function-or-method)
+  - [6.3. Constant Or Property](#63-constant-or-property)
+- [7. Describing hashes](#7-describing-hashes)
+- [8. Tags](#8-tags)
+  - [8.1.  @api](#81-api)
+  - [8.2.  @author](#82-author)
+  - [8.3.  @category [deprecated]](#83-category-deprecated) 
+  - [8.4.  @copyright](#84-copyright)
+  - [8.5.  @deprecated](#85-deprecated)
+  - [8.6.  @example](#86-example)
+  - [8.7.  @global](#87-global)
+  - [8.8.  @internal](#88-internal)
+  - [8.9.  @license](#89-license)
+  - [8.10. @link [deprecated]](#810-link-deprecated)
+  - [8.11. @method](#811-method)
+  - [8.12. @package](#812-package)
+  - [8.13. @param](#813-param)
+  - [8.14. @property](#814-property)
```

Comment:
I'd vote to add these too, there is a similar implementation in Yii framework: http://www.yiiframework.com/doc-2.0/guide-concept-properties.html https://github.com/yiisoft/yii2/blob/078b922114e5ba3481487a80f28769e69713ea33/framework/base/Object.php#L130-L163